content/maxwell3_fdem/emi_tutorial/amplitudeandphase.rst

```diff
@@ -0,0 +1,129 @@
+.. _amplitudeandphase:
+
+Amplitude and phase
+===================
+
+.. figure:: ./images/Concepts_3loops_only_K.png
+   :align: center
+   :scale: 60%
+   :name: Concepts_3loops_only_K
+
+   Conceptual diagram for 3-loops system.
+
+Based upon the induced current in the body (Loop2), in :ref:`induced_currents_body`, here we consider amplitude and phase of the secondary magnetic field (:math:`H^s`) normalized by primary magnetic field (:math:`H^p`), :math:`\frac{H^s}{H^p}`, measured at Rx (Loop3). For this, we first consider the primary magnetic field at body (Loop2), :math:`H^p_2`, resulting from a current, :math:`i^p`, flowing in the Loop1, which can be expressed as
+
+.. math::
+	H^p_2(t) = K_{12} i^p(t),
+
+where :math:`i^p = I^p cos(\omega t)` and :math:`K_{12}` is coefficint depending on the geomtry of the system, the area, and number of turns of the primary coil, and attenuation of the wave between Loop1 and Loop2.
+
+Effectively, through Faraday's law, :math:`\mathcal{E} = -\imath \omega \Phi^p`, volatage will be generated, which generates induced currents (this has been done in :ref:`induced_currents_body`):
```

Comment:
Here, it might not be clear that you are talking about a voltage and currents generated within the loop. I would suggest specifying that this is within loop2 ("a voltage will be generated within loop2"). 
(and spell check in this doc too --> in sublime: view / spellcheck)